docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3"
+
+services:
+  alice:
+    build: .
+    image: dhxdocker/datahighway:latest
+    ports:
+      - 30333:30333
+      - 9944:9944
+      - 9933:9933
+    volumes:
+      - ./src/chain-definition-custom:/dhx/src/chain-definition-custom
```

Comment:
i don't see why we're using this folder for the volume

Reply:
it's outdated, in the current compose file we mount the ./scripts folder and set service specific startup scripts for alice/bob:

`    volumes:
      - ./scripts:/dhx/scripts
    entrypoint:
      - ./docker-entrypoint-alice.sh
`